test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "PSDiagnostics cmdlets tests" -Tag "CI", "RequireAdminOnWindows" {
+    BeforeAll {
+        $OriginalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if ( -not $IsWindows ) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+    }
+    AfterAll {
+        $Global:PSDefaultParameterValues = $OriginalDefaultParameterValues
+    }
+
+    Context "Test for Enable-PSTrace cmdlet" {
+        it "Should enable Analytic logs for Microsoft-Windows-PowerShell." {
+            Enable-PSTrace -Force
```

Comment:
If the test is "should enable" then before the test the log should be in disabled state. 
Also we should restore default state. Perhaps it male sense combine the test and tests below.

Reply:
Addressed